helix-rest/src/main/java/org/apache/helix/rest/metadatastore/accessor/ZkRoutingDataReader.java

```diff
@@ -36,8 +36,8 @@
 import org.apache.helix.zookeeper.zkclient.exception.ZkNoNodeException;
 import org.apache.zookeeper.Watcher;
 
-
-public class ZkRoutingDataReader implements MetadataStoreRoutingDataReader, IZkDataListener, IZkChildListener, IZkStateListener {
+public class ZkRoutingDataReader
+    implements MetadataStoreRoutingDataReader, IZkDataListener, IZkChildListener, IZkStateListener {
```

Comment:
Do you have latest helix-style-intellij.xml imported in your IDE? This format shouldn't be changed if you have the style import. It would be better to keep the original code that is not your change.

Reply:
I'm using helix-style.xml as suggested by this doc: https://cwiki.apache.org/confluence/display/HELIX/Coding+Style . The linked doc come from https://iwww.corp.linkedin.com/wiki/cf/pages/viewpage.action?pageId=209597012 . helix-style-intellij.xml was not mentioned anywhere as far as I know.

Reply:
Turns out that the onboarding doc links to an incorrect coding style page. Correcting it.